### DISCUSSION TITLE
Cannot access github main page from yesterday

### DISCUSSION THREAD

**Discussion by Junchao-Mellanox** [community/community#19737] [Other Feature Feedback, Questions, & Ideas]
Hi,

I have been using company VPN to access github. It worked file until yesterday. I asked company IT to check the issue, they suspected that our public IP has been banned by github. I suppose it is not safe to expose public IP of my company here. So, could you please give me an email so that I can share some information via email thread?

Thanks!

**Comment by thuongtruong1009** [comment]
Hi @Junchao-Mellanox 
***Method*** ***1***:  You can use Cloudflare 1.1.1.1 to hide your company information and be able to access Github
***Method*** ***2***: You can use Ubuntu or Linux virtual machine to access is also a good idea
***Method*** ***3***: You can view your Github profile via mobile web or mobile app
You can rest assured that Github for Microsoft is maintained and developed, so they are safe for your information.
Hope useful for you 🥰

**Reply by rilysh** [reply]
Using a vm won't help much as it's still located at the same public IP if they run on that same IP address.

**Comment by huabin** [comment]
@Junchao-Mellanox 

In your case, except what they told you above, you do need to contact GitHub support (https://support.github.com/) and share the information with them.

**Comment by sinkaroid** [comment]
https://support.github.com/request/account?tags=dotcom-footer%2Chubberfy_account
